### DISCUSSION TITLE
Modify the retention period for CDRs in EFK stack

### DISCUSSION THREAD

**Discussion by caprivm** [magma/magma#8563] [Q&A]
Hi All!,

I'm trying to modify the retention period from the Kibana UI but the options are related to Hot, Warm and Cold data, nor for all the data that is store in Elastic. So, how to change the retention period in orchestrator (both cases, Docker and Kubernetes implementation)?.

The magma CDRs are generated through EKF, but the default storage period is 2 months, we want to change it to 7 days! Thanks for any help!

**Comment by themarwhal** [comment]
We don't seem to have documentation for this, but we do have a `elasticsearch_retention_days` exposed as a configurable value. (See https://sourcegraph.com/github.com/magma/magma/-/blob/orc8r/cloud/deploy/orc8r_deployer/docker/root/data/vars.yml?L403)

This value then gets applied into terraform like this: https://github.com/magma/magma/blob/e4088e04e42cc3793bbf88c3bcf71755fa3d7973/orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf#L164-L199

Let me know if that helps! (And if it does, we welcome any documentation contributions to help future users :) )
